_includes/tutorials/session-windows/kstreams/code/build.gradle

```diff
@@ -34,9 +34,9 @@ dependencies {
     implementation "org.apache.avro:avro:1.10.1"
     implementation "org.slf4j:slf4j-simple:1.7.30"
     implementation "org.apache.kafka:kafka-streams:2.7.0"
-    implementation('io.confluent:kafka-streams-avro-serde:5.5.1') {
+    implementation('io.confluent:kafka-streams-avro-serde:6.1.0') {
         // this transitive dependency exclusion
-        // is needed as kafka-clients:5.5.1-ccs has a version
+        // is needed as kafka-clients:6.1.0-ccs has a version
```

Comment:
@bbejeck this test is NOT failing, unlike the other 2 tests that have similar exclusions

Reply:
The exclusion probably isn't needed anymore, but let's leave this for now and do a separate PR

Reply:
Ok, will leave it in.  Thanks.